tests/sites_framework/models.py

```diff
@@ -9,7 +9,6 @@ class AbstractArticle(models.Model):
     title = models.CharField(max_length=50)
```

Comment:
Does this actually identify a regression in 3a47d42fa33012b2156bf04058d933df6b3082d2? The check error `CustomArticle.on_site: (sites.E001) CurrentSiteManager could not find a field named 'sites'.` seems to suggest that model's override of the parent manager isn't working. \cc @loic